requirements.txt

```diff
@@ -10,10 +10,14 @@ Flask-GraphQL==1.4.1
 google-api-python-client==1.6.7
 graphene-sqlalchemy==2.0.0
 graphene==2.1
+imgkit==1.0.1
 MarkupSafe==1.0
 more-itertools==4.1.0
+matplotlib==3.0.0
```

Comment:
You could delete this dependency since it is unused.

Reply:
Thanks @lennykioko . Done already.